package ErrorRet;

message ErrorRet {
	required uint32 errorret		 	= 1;
	required string errormsg			= 2;
	required string requestmsg			= 3;
}